install.sh

```diff
@@ -9,19 +9,38 @@ DIR=${1-lib}
 VERSION=${2-latest}
 TAG="$LATEST_BASHUNIT_VERSION"
 
+function build_and_install_beta() {
+  echo "> Downloading non-stable version: 'beta'"
+  git clone --depth 1 --no-tags https://github.com/TypedDevs/bashunit temp_bashunit 2>/dev/null
+  cd temp_bashunit
+  ./build.sh >/dev/null
+  cd ..
+  cp temp_bashunit/bin/bashunit ./
+  sed -i -e 's/BASHUNIT_VERSION=".*"/BASHUNIT_VERSION="(non-stable) beta"/g' bashunit
+  rm -rf temp_bashunit
+}
+
+function install() {
+  if [[ $VERSION != 'latest' ]]; then
+    TAG="$VERSION"
+    echo "> Downloading a concrete version: '$TAG'"
+  else
+    echo "> Downloading the latest version: '$TAG'"
+  fi
+
+  curl -L -O -J "https://github.com/TypedDevs/bashunit/releases/download/$TAG/bashunit" 2>/dev/null
```

Comment:
Would it make senses to add header on the `curl` to track the version requested as well as the OS that is using?

Reply:
https://reqbin.com/req/c-ea0d5rlb/curl-send-header-example